### ISSUE TITLE
Add --skip-photos option

### ISSUE THREAD

**Issue by manumora** [icloud-photos-downloader/icloud_photos_downloader#401]
## Summary
Add config option:  --skip-photos

## Context
Since I don't have too much space in my iCloud account, I would like to keep my photos but download all my videos to my NAS due to videos are heavier. Current options allow us the opposite, to download photos and skip videos but I wanted to do download only my videos. As far I know, it is not possible.

I have been studying the code and seems to add this new option is easy because is similar to --skip-videos

If you agree, I can add this option and generate the Pull Request.

Thank you!

**Comment by AndreyNikiforov** [comment]
good idea. what do you think the behavior should be for live photos that have image and video? for either of skip flags of course.
i assume we'll also need to a check that both skip flags are not allowed at the same time from cli

**Comment by manumora** [comment]
> what do you think the behavior should be for live photos that have image and video?

I am not sure but it could work in the same way, if skip photos is enabled, live-photos videos will be downloaded.

> i assume we'll also need to a check that both skip flags are not allowed at the same time from cli

Sure, it would not make sense to enable both skip-photos and skip-videos at the same time.

**Comment by boredazfcuk** [comment]
Any chance I can piggy back one on this? A --skip-screenshots which ignores all .png files?

**Comment by SHASHWATAA** [comment]
Hey I had the same problem, what I did was create a smart album with all videos and downloaded that.

**Comment by MatrixDai** [comment]
The smart album with name `Videos` is predefined in the `pyicloud_ipd` lib, to download videos only, just specify the argument `-a Videos`, of course do not specify the `--skip-videos` at same time.
But the videos will not get deleted after download, so I will try to add a such feature.

**Comment by ipaqmaster** [comment]
Had to cancel with 29000/37900 downloaded last night and the script is insisting on downloading every single file again. Could it just be a case of quickly checking the filename-to-be for existence on the filesystem already? It may not speed things up as desired but it will avoid redownloading and re-writing so much data over existing snapshots again

**Comment by nicojx** [comment]
I think it would be great to have this option, and I think it would make sense to download (and then delete) only genuine videos, without live photo videos.

**Comment by AndreyNikiforov** [comment]
fixed in [v1.30.0](https://github.com/icloud-photos-downloader/icloud_photos_downloader/releases/tag/v1.30.0)
